layouts/shortcodes/capture.html

```diff
@@ -1,7 +1 @@
-{{ $_hugo_config := `{ "version": 1 }`}}
-{{- $id := .Get 0 -}}
-{{- if not $id -}}
-{{- errorf "missing id in capture" -}}
-{{- end -}}
-{{- $capture_id := printf "__cid_%s" $id -}}
-{{- .Page.Scratch.Set $capture_id .Inner -}}
\ No newline at end of file
+{{ warnf "Invalid shortcode in %q" .Page.Path }}
```

Comment:
@sftim, What do you think about adding more information to the warning.
I tested this shortcode with the warning. It would be helpful to provide the capture_id in the warning message.
I could see leaving lines 1-6 and adding the `capture_id` to the warning message?
Also, add the <lang> prefix to identify with the page path:

```
{{ $_hugo_config := `{ "version": 1 }`}}
{{- $id := .Get 0 -}}
{{- if not $id -}}
{{- errorf "missing id in capture" -}}
{{- end -}}
{{- $capture_id := printf "capture %s" $id -}}
<!--
{{- .Page.Scratch.Set $capture_id .Inner -}}
-->
{{ warnf "Invalid shortcode: in %s, %q" $capture_id (relLangURL .Page.Path) }}
```